03-nodejs/00-learn-nodejs-basics/01-content-negotiation/requestForAuthorInfo.js

```diff
@@ -1 +1,24 @@
-// Your code here
\ No newline at end of file
+// Your code here
```

Comment:
Also be consistent, if you use header with your authority information, provide it in each file